### ISSUE TITLE
Customizable Interface (or at least being able to move the "layers window")

### ISSUE THREAD

**Issue by jjhaggar** [LibreSprite/LibreSprite-proposals#45]
##### Is your feature request related to a problem? Please describe
I often find myself feeling like im kind of cramped with the layer system when I'm trying to work on larger canvases and lots of layers.

##### Describe the solution you'd like
I thought it would be super awesome if that layer window could be moved to the left or right of the work space. As well as the other elements of the interface (toolbar, palette, etcetera).
![image](https://user-images.githubusercontent.com/4078392/143719552-c01386ab-d76d-4bfc-a200-4f1eaabe6008.png)

##### Describe alternatives you've considered
Another way of solving this could be having dockable/undockable windows (or dialogs, as they call them in Gimp https://docs.gimp.org/2.8/en/gimp-concepts-docks.html), but it seems much harder to me... Then again, I'm not a developer, so maybe this is easier and it's the way to go? I don't know ¯\\\_(ツ)\_/¯

##### Additional context
Organizing the windows of the program (and even being able to save different windows distributions, often called workspaces) is a common thing in art related software. For example, you can do it in Blender, OpenToonz, Gimp, etc.
Blender https://www.youtube.com/watch?v=NS7HyFCclII
OpenToonz https://www.youtube.com/watch?v=T-Qy5u3iG_E
Gimp https://www.youtube.com/watch?v=CKM1Vq0Paa4

**Comment by jjhaggar** [comment]
(This feature was requested by _**Dances with Madness**_ in the Discord server)

**Comment by therealrobster** [comment]
I would like to chime in on this.  I am testing Libresprite now (as an owner and regular user of Aseprite).  I love you've kept it open source so want to support the effort.  So first of all thank you.

In regard to above, I agree.  In Aseprite there is an option in the Onion Skin options that allows left, right and bottom for docking of the layers.  When animating, sure.  The layers at the bottom makes sense.  When drawing though, on the right or left makes more sense.  

I trust the feedback helps.

**Comment by Zughy** [comment]
I completely agree with you all, the "only" problem is that before implementing such a feature we need to refactor the UI code, as right now is... well, not great
